File: compare_triplets.rb
# https://www.hackerrank.com/challenges/compare-the-triplets/problem

def compareTriplets(a, b)
  alice = 0
  bob = 0

  a.length.times do |i|
    if a[i] > b[i]
        alice += 1
    elsif a[i] < b[i]
        bob += 1
    end
  end

  [alice, bob]
end

File: compare_triplets.py
# https://www.hackerrank.com/challenges/compare-the-triplets/problem

def compareTriplets(a, b):
    alice = 0
    bob = 0
    for i in range(len(a)):
        if a[i] > b[i]:
            alice = alice + 1
        elif a[i] < b[i]:
            bob = bob +1

    return [alice, bob]
